RecoVertex/BeamSpotProducer/BuildFile.xml

```diff
@@ -16,6 +16,7 @@
 <use   name="clhep"/>
 <use   name="rootcore"/>
 <use   name="rootminuit2"/>
```

Comment:
I just tested and removing `<use name="rootminuit2"/>` does not appear to cause any adverse effects.
@davidlange6 do you want to me to update the pull request with this removed? Or do you want that as a seperate PR?